packages/palette/src/elements/Pill/tokens.ts

```diff
@@ -50,19 +50,23 @@ export const PILL_VARIANTS: Record<PillVariant, Record<PillState, any>> = {
       border-radius: 20px;
       height: 40px;
       padding: 0 ${themeGet("space.2")};
+      color: ${themeGet("colors.black60")};
```

Comment:
This isn't correct according to the Figma spec

Reply:
Spec = https://www.figma.com/file/m6gDpKHEWDbYJyrwsVZDBr/Artsy-3.0-Design-System?node-id=1020%3A781

Reply:
But in [this](https://www.figma.com/file/0svJJjlcrCFBkVYzagKP3Z/FX-v3-Surfaces?node-id=4128%3A1220) spec default text color is `black60`